pkg/command/server.go

```diff
@@ -144,7 +170,7 @@ func Server(cfg *config.Config) *cli.Command {
 				}
 
 				gr.Add(func() error {
-					//return server.Run()
+					server.ListenAndServe()
```

Comment:
```go
if err := server.listenAndServer; err != nill {
logger.Error(err)
}

Reply:
and return the err - afaik ....


if we do not log this err here we never know why the server is not starting (port already in use e.g.)

Reply:
the ListenAndServe in glauth currently does notr return an error ...

Reply:
oh - we shall fix this then ....

Reply:
fixed with https://github.com/glauth/glauth/pull/126